examples/cli/env.roc

```diff
@@ -5,6 +5,8 @@ app "env"
 
 main : Task {} []
 main =
+    f = ""
```

Comment:
Should this be removed @ayazhafiz?

Reply:
🤦🏼 will send a PR to remove. Sorry about that

Reply:
No problem, stuff like that happens to everyone :)